app/views/application/_sidebar.html.erb

```diff
@@ -16,7 +16,7 @@
 
   <%= yield :sidebar %>
 
-  <% if current_user %>
+  <% unless current_user.try(:teams).blank? %>
```

Comment:
You could use `try!` here instead of `try` since it's safer to use on the view and easier to track bugs.